layouts/_shortcodes/sdlc.html

```
@@ -0,0 +1,52 @@
+			<th><a href="https://about.gitlab.com/stages-devops-lifecycle/{{.key}}">
+				<img src="/images/sdlc-icons/{{.key}}.svg" height="32" width="32"><br>
+				{{.display_name}}</a></th>
+		{{- end }}
+		</tr>
+	</thead>
+	<body>
+		{{- range $marketingStages -}}
+			<td>{{- range .categories -}}
+				{{- $url := partial "categories/best-link" . -}}
+				{{- if $url -}}
+				{{ printf "<a href=\"%s\">" $url |safeHTML }}<img src="/images/maturity/{{.maturity}}.svg" height="16" width="16"> {{ .name }}</a><br>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)